app/models/amendments/articulated_amendment.rb

```diff
@@ -5,7 +5,11 @@ class ArticulatedAmendment < Amendment
     has_one :articulated, foreign_key: 'amendment_id'
     delegate :section, to: :articulated, allow_nil: true
 
-    def allow_articulateds?
+    def articulated?
+      !articulated.nil?
```

Comment:
Try to not negate a negative question method. You can ask for `articulated` or, if you want to be more explicit, `articulated.present?`

Reply:
Solved